lib/Launcher.js

```diff
@@ -61,10 +61,13 @@ class Launcher {
   static async launch(options) {
     options = Object.assign({}, options || {});
     let temporaryUserDataDir = null;
-    const chromeArguments = [].concat(DEFAULT_ARGS);
+    const chromeArguments = [];
+    if (!options.disableDefaultArgs)
```

Comment:
in chrome-launcher we're calling it `ignoreDefaultArgs`. i do think the `ignore` verb is more appropriate, especially when we're talking about a collection of things, and not a singular item.

Reply:
Done.